less/admin/UsersListPage.less

```diff
@@ -0,0 +1,118 @@
+.UserListPage {
+  // Pad bottom of page to make nav area look less squashed
+  margin-bottom: 24px;
```

Comment:
In most other pages where we need bottom spacing, we use `padding-bottom: 30px;` could we use that here for consistency ?

Reply:
Sure, but I'd rather use a multiple of 8px to match the [widespread standard](https://spec.fm/specifics/8-pt-grid) used across the web, and [specifically recommended by Apple](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/image-size-and-resolution/#designing-high-resolution-artwork) to prevent odd blurring caused by scaling.

Reply:
That's so interesting! I've learned something new. Fine by me, though most of our codebase doesn't follow that standard, so I'm not sure if it would make a difference.